aws/ansible/templates/ote/config.edn.j2

```diff
@@ -44,4 +44,8 @@
 	       :detected-changes-recipients "{{vault_pre_notices_detected_changes_recipients}}"
                :laundry-url {{laundry_url}} }
  :no-gtfs-update-for-operators {{no_gtfs_update_for_operators}}
+
+:netex {:chouette-path "{{netex-chouette-path}}"
+ :conversion-work-path "{{netex-conversion-work-path}}"
+ :bucket "{{netex-bucket}}"}
```

Comment:
Intention

Reply:
Will fix in next PR - indentation doesn't look like this on my local file so have to see what's wrong.